deploy/01_manager.ts

```diff
@@ -21,6 +21,6 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 };
 
 func.id = "deploy_manager";
-func.tags = ["Manager"];
+func.tags = ["Manager", "core"];
```

Comment:
any ideas why this was previously not marked as a core contract?

Reply:
Not sure why, but `core` should also be added in 04_* && 05_* deploy script.

Reply:
Thanks man.